packages/docs/src/components/DocsApi/api-docs-helpers.ts

```diff
@@ -76,14 +82,40 @@ const getApiTableProps = (
   return api
 }
 
+const mergeApiTableEvents = (
+  componentEvents: Record<string, EventOptionsCompiled> = {},
+  manualEvents: Record<string, ManualEventApiOptions> = {},
+) => {
+  const merged: Record<string, ManualEventApiOptions | EventOptionsCompiled> = {}
+  const uniqueManualEvents = { ...manualEvents }
+
+  for (const componentEventName in componentEvents) {
+    // handle cases when we use events with such pattern `update:myPropName`
+    // it's needed since all events from manualOptions transformed to kebab-case
+    // in the case if need both events `click-day` and `click:day` in one component, mergeApiTableEvents and translations logic should be reworked
+    const kebabComponentEventName:string = kebabCase(componentEventName)
```

Comment:
It would be nice to have tests for all these complex things. I mean for the whole api-docs-helpers file. 

I made an issue https://github.com/epicmaxco/vuestic-ui/issues/1026

But in this PR you can make tests for this mergeApiTableEvents method if you want.